### PR TITLE
Support italic style in printstyled.

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Library changes
 * `Dict` can be now shrunk manually by `sizehint!` ([#45004]).
 * `@time` now separates out % time spent recompiling invalidated methods ([#45015]).
 * `@time_imports` now shows any compilation and recompilation time percentages per import ([#45064]).
+* `printstyled` now support italic output ([#?]).
```
Comment (KristofferC) [inline NEWS.md:60]:
```suggestion
* `printstyled` now supports italic output ([#?]).
```

?

**Review by KristofferC — APPROVED** [review]

**Review by vtjnash — CHANGES_REQUESTED** [review]
We did not include this because it does not reliably make it italic, except on monochrome monitors. On color monitors, it may invert the colorscheme instead:

macOS Terminal running tmux:
image

macOS Terminal not running tmux:
image

https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters

**Comment by vtjnash** [comment]
Note that even this support might be quite recent, since people were writing blogs about it not working within the past few years: https://weibeld.net/terminals-and-shells/italics.html

**Comment by fredrikekre** [comment]
Rebased. I included a note about terminal support in the docstring.

**Review by vtjnash — APPROVED** [review]

**Comment by tecosaur** [comment]
It's worth noting (and I think short of the docs a comment here has a chance of being seen at least) that italic support can be checked for in terminfo, and so the best practice would be adapting the printing behavior based on the terminal capabilities.

**Pull request merged**